srcalc/src/test/java/gov/va/med/srcalc/web/view/VariableEntryTest.java

```diff
@@ -67,4 +69,31 @@ public final void testRetrievedCardiacAge()
     	
     	assertEquals(expected, entry.getDynamicValues());
     }
+    
+    @Test
+    public final void testWithRetrievedLabs()
+    {
+        final List<AbstractVariable> vars = SampleModels.sampleVariableList();
+        final Patient patient = SampleCalculations.dummyPatientWithLabs(1);
+        final VariableEntry entry = VariableEntry.withRetrievedValues(vars, patient);
+        final HashMap<String, String> expected = new HashMap<>();
+        
+        expected.put(vars.get(1).getKey(), String.valueOf(patient.getAge()));
+        expected.put(vars.get(2).getKey(), patient.getGender());
+        expected.put(vars.get(5).getKey(), VariableEntry.SPECIAL_NUMERICAL);
+        
+        final String numericalString = VariableEntry.SEPARATOR + VariableEntry.SPECIAL_NUMERICAL;
+        final String dateString = numericalString + VariableEntry.SEPARATOR + VariableEntry.RETRIEVAL_STRING;
```

Comment:
Let's just make these constants private now and this test can just test the get/set interface instead of the implementation detail.